dot-analysis.cabal

```diff
@@ -19,7 +19,7 @@ maintainer:         guillaume.allais@ens-lyon.org
 -- A copyright notice.
 -- copyright:
 -- category:
-extra-source-files: CHANGELOG.md
+-- extra-source-files: CHANGELOG.md
```

Comment:
We can just get rid of it, it was automatically inserted by `cabal init` but is (currently) useless
```suggestion
```